fe/fe-core/src/main/java/org/apache/doris/nereids/trees/copier/LogicalPlanDeepCopier.java

```diff
@@ -412,4 +426,24 @@ private void updateReplaceMapWithOutput(Plan oldPlan, Plan newPlan, Map<ExprId,
         }
     }
 
+    private Plan updateOperativeSlots(LogicalCatalogRelation oldRelation, LogicalCatalogRelation newRelation) {
+        List<Slot> oldOperativeSlots = oldRelation.getOperativeSlots();
+        List<Slot> newOperativeSlots = new ArrayList<>(oldOperativeSlots.size());
+        int outputSize = oldOperativeSlots.size();
+        for (Slot opSlot : oldOperativeSlots) {
+            int idx;
+            for (idx = 0; idx < outputSize; idx++) {
+                if (opSlot.equals(oldRelation.getOutput().get(idx))) {
+                    newOperativeSlots.add(newRelation.getOutput().get(idx));
+                    break;
+                }
+            }
```

Comment:
maybe use `ExpressionDeepCopier.INSTANCE.deepCopy(opSlot, context)` is a better way